src/components/RandomProductsBanner/RandomProductsBanner.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+$product-item-default-img-height: 150px;
```

Comment:
Please, review all .scss files in components and see if you can move these variables to general _variables file. I am sure most of these heights and fontsizes will be reused, so move the reusable or that can be reused in future to variables.